WifiART/src/structs/UsageState.java

```diff
@@ -0,0 +1,25 @@
+package structs;
+
+public class UsageState {
+    // Parameters
+    private boolean retransmissionAllowed;
+    private boolean retentionExpires;
+    private int expireTime;
```

Comment:
Add a unit - is the time microseconds, milliseconds or seconds?

Reply:
Are you referring to the naming of the variable or are you suggesting that I provide a comment in the variable declaration explaining the units? Note that in the setter for this field, the units are specified as hours.

Reply:
The naming itself. i.,e this should expireTimeHours